CaImAn/caiman_run.py

```diff
@@ -37,55 +37,56 @@
 
 kwargs = {}
 params_doric = {}
+danse_parameters = {}
 
 try:
     for arg in sys.argv[1:]:
         exec(arg)
 except SyntaxError:
-    print("[intercept] One of the advanced settings is not of a python type [end]", flush=True)
+    print_to_intercept("One of the advanced settings is not of a python type")
```

Comment:
Should we define all the strings at the beginning of the document?

Reply:
yeah. I will do that when refactoring everything

Reply:
Then what do we clean here? Feels like refactoring and cleaning are actually the same task, or in cleaning, we don't do much 🤷‍♀️

Reply:
yeah true. I will do that then.